workspace/utilities/lib/iframe-copyright-288.xsl

```diff
@@ -59,13 +59,10 @@
 		
 		<xsl:variable name="computed-attr">
 			<add class="copy-288" />
-			<add src="{$url}" />
-			<add margin="0" />
-			<add border="0" />
-			<add scrolling="no" />
-			<add frameborder="no" />
-			<add width="{$w}" />
-			<add height="{number($h)}" />
+			<set src="{$url}" />
+			<set width="{$w}" />
+			<set height="{number($h)}" />
+			<set style="border:0;margin:0;overflow:hidden">
```

Comment:
missing ; at the end.
We should put space between each property to allow simple partial alteration with rem later.

Reply:
I do not want it to be removed. the extra ; is not needed.

Reply:
the extra; is needed if someone add a style attribute it will break.

Reply:
done in 3c2f3f1